rbc/remotejit.py

```diff
@@ -19,7 +19,7 @@
 # XXX WIP: the OmnisciCompilerPipeline is no longer omnisci-specific because
 # we support Arrays even without omnisci, so it must be renamed and moved
 # somewhere elsef
-from .omnisci_backend import OmnisciCompilerPipeline
+from .heavyai import OmnisciCompilerPipeline
```

Comment:
At some point (not necessarily here to keep this PR simple), we'll need to rename `OmnisciCompilerPipeline` as well.